src/main/java/com/esri/samples/scene/view_point_cloud_data_offline/README.md

```diff
@@ -0,0 +1,33 @@
+<h1>View point cloud data offline</h1>
+
+<p>Display local 3D point cloud data.</p>
+
+<p><img src="ViewPointCloudDataOffline.png"/></p>
+
+<h2>Use case</h2>
+
+<p>Point clouds are often used to visualize massive sets of sensor data such as lidar. The point locations indicate where the sensor data was measured spatially, and the color or size of the points indicate the measured/derived value of the sensor reading. In the case of lidar, the color of the visualized point could be the color of the reflected light, so that the point cloud forms a true color 3D image of the area.</p>
+
+<p>Point clouds can be loaded offline from scene layer packages (.slpk).</p>
```

Comment:
I think this could be better in additional information section: doesn't fit well in the use case section.